OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -153,6 +157,9 @@ public boolean isWebCourseSearchEnabled() {
         @SerializedName("SEARCH_BAR_ENABLED")
         private boolean mSearchBarEnabled;
 
+        @SerializedName("SUBJECT_DISOVERY_ENABLED")
```

Comment:
There is a typo here, its not according to config flag.
**SUBJECT_DISCOVERY_ENABLED**

Reply:
Ooops!!!